docs/contributing/where-should-my-code-live.md

```diff
@@ -0,0 +1,64 @@
+# Where should my code live?
+
+## Scripts
+
+Externally hosted third party scripts should always be loaded asynchronously. If possible, they should be loaded conditionally using JavaScript by injecting a script element into the head of the document.
+
+If a third party script needs to be loaded on every page, there are two ways it could be categorised: **low priority** or **high priority**.
+
+### Low priority scripts
+
+In the first instance, consider adding the script as a low priority script. These scripts are added to the bottom of the document `<body>` and loaded asynchronously. They will probably run after the main application JavaScript bundle, _but this is not guaranteed._
+
+An example is the [Google Analytics tracking script](https://developers.google.com/analytics/devguides/collection/analyticsjs/).
+
+They can be added to the `lowPriorityScripts` array in the [`packages/frontend/web/document.tsx`](https://github.com/guardian/dotcom-rendering/blob/master/packages/frontend/web/document.tsx) module.
+
+### High priority scripts
+
+High priority scripts are injected by JavaScript running in the `<head>` of the document. They are loaded asynchronously and executed in the order they are requested. 
+
+Examples include the [polyfill.io](https://polyfill.io) response and the main application JavaScript bundles.
+
+⚠️ **High priority scripts should be added sparingly.**
```

Comment:
I think it's probably worth saying at least two Dotcom team members need to approve any high priority scripts. (Or something similar, which effectively gatekeeps this to the Dotcom team.)

Reply:
Sounds good, I'll make it clearer that adding another high priority script is unlikely to be approved!